app/controllers/users_controller.rb

```diff
@@ -8,11 +8,14 @@ def create
     if user.valid?
       flash[:notice] = "#{user.name} is now logged in"
       redirect_to '/profile'
+    else
+      flash[:error] = "#{user.errors.full_messages.to_sentence}"
```

Comment:
Add flash message if the new form is not completely/correctly filled out. The flash message will say the specific fields that were incorrect.